src/utils.js

```diff
@@ -1,5 +1,10 @@
 const utils = {
 
+    getBasicAuthPassword: req => {
+        const content = req.get("Authorization").replace(/^Bearer /, '')
```

Comment:
Won't this be "Basic" instead of "Bearer"?

https://en.wikipedia.org/wiki/Basic_access_authentication#Client_side

Reply:
You're completely correct.... it was late and I was clearly too tired!